tests/ops/integration_tests/saas/test_zendesk_task.py

```diff
@@ -1,347 +1,122 @@
-            "via",
-            "created_at",
-            "updated_at",
-            "type",
-            "subject",
-            "raw_subject",
-            "description",
-            "priority",
-            "status",
-            "recipient",
-            "requester_id",
-            "submitter_id",
-            "assignee_id",
-            "organization_id",
-            "group_id",
-            "collaborator_ids",
-            "follower_ids",
-            "email_cc_ids",
-            "forum_topic_id",
-            "problem_id",
-            "has_incidents",
-            "is_public",
-            "due_at",
-            "tags",
-            "custom_fields",
-            "satisfaction_rating",
-            "sharing_agreement_ids",
-            "followup_ids",
-            "brand_id",
-            "allow_channelback",
-            "allow_attachments",
-        ],
-    )
-
-    assert_rows_match(
-        v[f"{dataset_name}:ticket_comments"],
-        min_size=1,
-        keys=[
-            "id",
-            "type",
-            "author_id",
-            "body",
-            "html_body",
-            "plain_body",
-            "public",
-            "attachments",
-            "audit_id",
-            "via",
-            "created_at",
-            "metadata",
-        ],
-    )
-
-    # verify we only returned data for our identity email
-    assert v[f"{dataset_name}:users"][0]["email"] == zendesk_identity_email
-    user_id = v[f"{dataset_name}:users"][0]["id"]
-
-    assert v[f"{dataset_name}:user_identities"][0]["value"] == zendesk_identity_email
-
-    for ticket in v[f"{dataset_name}:tickets"]:
-        assert ticket["requester_id"] == user_id
-
-    for ticket_comment in v[f"{dataset_name}:ticket_comments"]:
-        assert ticket_comment["author_id"] == user_id
-
-
-@pytest.mark.integration_saas
-@pytest.mark.integration_zendesk
-@pytest.mark.asyncio
-async def test_zendesk_erasure_request_task(
-    db,
-    policy,
-    erasure_policy_string_rewrite,
-    zendesk_connection_config,
-    zendesk_dataset_config,
-    zendesk_erasure_identity_email,
-    zendesk_create_erasure_data,
-) -> None:
-    """Full erasure request based on the Zendesk SaaS config"""
-
-    masking_strict = CONFIG.execution.masking_strict
-    CONFIG.execution.masking_strict = False  # Allow Delete
-
-    privacy_request = PrivacyRequest(
-        id=f"test_zendesk_erasure_request_task_{random.randint(0, 1000)}"
-    )
-    identity = Identity(**{"email": zendesk_erasure_identity_email})
-    privacy_request.cache_identity(identity)
-
-    dataset_name = zendesk_connection_config.get_saas_config().fides_key
-    merged_graph = zendesk_dataset_config.get_graph()
-    graph = DatasetGraph(merged_graph)
-
-    v = await graph_task.run_access_request(
-        privacy_request,
-        policy,
-        graph,
-        [zendesk_connection_config],
-        {"email": zendesk_erasure_identity_email},
-        db,
-    )
-
-    assert_rows_match(
```

Comment:
I removed these from the draft but we can add them back in later or make these field checks part the assertions in the base access request